README.md

```diff
@@ -0,0 +1,32 @@
+BaseBall Card Tracker Apps
+=
+
+Baseball Card Tracker (BBCT) is a simple database app for your baseball card collection. You can add information about the cards in your collection, view details about individual cards, and filter the list of cards according to a limited set of criteria. BBCT helps you organize your collection and quickly look up which cards you own. Feel free to email your feedback and comments or visit the project website.
+
+
+Product features
+-
+
+* Store information about your baseball cards, including card brand, publication year, card number, current monetary value, number of cards you own, and player's name, team, and position.
+* Filter list of cards using simple criteria, including card year, card number, both year and number, player's name, or team
+* Update monetary value and number of copies that you own of a particular card
+
+Latest updates
+-
+
+**What's new in version 0.6.8.1**
+* Version 0.6.8.1
+* Fix bug that prevents editing cards
+
+Contributing
+-
+
+If you are interested in contributing to BBCT, please join our [Google Group](https://groups.google.com/forum/?hl=en#!forum/bbct). Then [download and install the desktop app](https://sourceforge.net/projects/bbct/files/latest/download) and/or the Android app from Google Play. Read [Getting Started](https://github.com/BaseballCardTracker/bbct/wiki/Getting-Started) for instructions about what tools to install and how to get the code. Finally, browse the open issues to find a task that interests you. When you are ready to submit changes, make a [Pull Request](https://help.github.com/en/articles/about-pull-requests) for a code review. If you have any questions, please [post the our Google Group](https://groups.google.com/forum/?hl=en#!newtopic/bbct), Make sure you already join the Group before asking a question.
+
+**Here is some link to help contribute**
```

Comment:
Put "Additional Links" for this title.

Reply:
I see that you marked this comment as "resolved", but I don't see the requested change. Please let me know if you have any questions.